markdown/developers/guidelines/requirements/keywords.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Keywords
+subtitle: Excellent documentation and GitHub repository keywords.
```

Comment:
I find this confusing, the title is documentation but you don't actually refer to any docs other than it should be execellent?

Reply:
Could rephrase with `findability` or something, and side-note: have we given up with bio.tools?

Reply:
> I find this confusing, the title is documentation but you don't actually refer to any docs other than it should be execellent?

That is correct. It should be excellent. Not sure what else to say?